plugins/org.yakindu.sct.ui.editor/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,4 +1,14 @@
 eclipse.preferences.version=1
+org.eclipse.jdt.core.codeComplete.argumentPrefixes=
```

Comment:
Please remove JDT preferences from the project since we use the settings defined in oomph setup file

Reply:
These are only to enable Eclipse to understand the comments `formatter:on` and `formatter:off`, because without it the statechart decl template will be made unreadable by the auto formatter...

Reply:
you should add this settings to oomph setup then, formatter preferences should not be project specific